pkg/kor/pvc.go

```diff
@@ -0,0 +1,119 @@
+package kor
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+)
+
+func retreiveUsedPvcs(kubeClient *kubernetes.Clientset, namespace string) ([]string, error) {
+	pods, err := kubeClient.CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		fmt.Printf("Failed to list Pods: %v\n", err)
+		os.Exit(1)
+	}
+	var usedPvcs []string
+	// Iterate through each Pod and check for PVC usage
+	for _, pod := range pods.Items {
+		for _, volume := range pod.Spec.Volumes {
+			if volume.PersistentVolumeClaim != nil {
+				usedPvcs = append(usedPvcs, volume.PersistentVolumeClaim.ClaimName)
+			}
+		}
+	}
+	return usedPvcs, err
+}
+
+func calculatePvcDifference(usedConfigMaps []string, configMapNames []string) []string {
```

Comment:
the variable names mention `ConfigMaps` but the function is about PVCs, this might be a copy/paste issue, maybe we can just call them `usedObjectNames` and `allObjectNames`?